### DISCUSSION TITLE
Preferred Method for Missing Cities?

### DISCUSSION THREAD

**Discussion by borth79** [dr5hn/countries-states-cities-database#302] [Ideas]
What is the preferred method for contact in regards to missing locations?

Suggestion: 
* Add the preferred method to report missing locations in the readme file.
* Add a form to help automate the submission process
* On the mission location form, add a map allowing users to help identify the location coordinates

**Comment by dr5hn** [comment]
@borth79 Currently preferred method is the "Issues" tab in Github.

A good tool to have. Thanks for the suggestions.
